src/App.js

```diff
@@ -0,0 +1,92 @@
+import './App.css';
+import { Component } from 'react';
+import Transactions from './Components/Transactions';
+import Operations from './Components/Operations';
+import axios from 'axios';
+import { BrowserRouter as Router, Redirect, Route } from 'react-router-dom';
+import Breakdown from './Components/Breakdown';
+import React from 'react'
+import { ThemeProvider, createMuiTheme } from '@material-ui/core/styles'
+import Navbar from './Components/Navbar';
+
+const theme = createMuiTheme({
```

Comment:
try to separate your styles into a styles.js file. Otherwise, your components get too crowded